index.html

```diff
@@ -12,39 +12,36 @@
 <body>
   <!-- Section 1: white portion/racoon -->
   <section>
-    <hr class="line"></hr>
-    <article id="racoonsection">
+    <hr class="line">
+    <article id="what-is-topcoat" class="what-is-topcoat-style">
```

Comment:
ok, I will take off the styles now...  the id's will have the same naming convention as the class names